bridges/otellogr/logsink.go

```diff
@@ -61,6 +63,11 @@ import (
 	"go.opentelemetry.io/otel/log/global"
 )
 
+const (
+	// exceptionMessageKey is the key used for the error message.
+	exceptionMessageKey = "exception.message"
```

Comment:
Use `semconv.ExceptionMessageKey`?

https://github.com/open-telemetry/opentelemetry-go/blob/main/semconv/v1.27.0/attribute_group.go#L3213

Reply:
Also it should always be a string. Therefore we should call `err.Error()` instead of `convertValue(err)`.